src/platform/plugins/private/links/common/content_management/v1/types.ts

```diff
@@ -59,10 +62,16 @@ export type Link = DashboardLink | ExternalLink;
 
 export type LinksLayoutType = typeof LINKS_HORIZONTAL_LAYOUT | typeof LINKS_VERTICAL_LAYOUT;
 
+export type LinksTextOverflowType =
+  | typeof LINK_TEXT_OVERFLOW_ELLIPSIS
+  | typeof LINK_TEXT_OVERFLOW_WRAP
+  | typeof LINK_TEST_OVERFLOW_SCROLL;
```

Comment:
I don't think the "scroll" should be a separate type here - ultimately, the difference between wrap and scroll is just whether or not the layout is vertical or horizontal, so we can make the distinction in the UI based on that, instead. So we can get away with just:

```suggestion
export type LinksTextOverflowType =
  | typeof LINK_TEXT_OVERFLOW_ELLIPSIS
  | typeof LINK_TEXT_OVERFLOW_WRAP;
```

This prevents the layout + `LINK_TEXT_OVERFLOW_WRAP / LINK_TEST_OVERFLOW_SCROLL` settings from coming out of sync